tests/Http/UserTest.php

```diff
@@ -1,67 +0,0 @@
-<?php
-
-namespace Tests\Http;
-
-use Rogue\Models\User;
-use Rogue\Services\Phoenix;
-use Tests\BrowserKitTestCase;
-use DoSomething\Gateway\Northstar;
-
-class UserTest extends BrowserKitTestCase
```

Comment:
Would it make sense to mark these as incomplete instead of removing them?

Reply:
I've got a PR ready to push up after merging this, but otherwise I definitely would!